apps/system/js/statusbar.js

```diff
@@ -431,6 +440,12 @@
         case 'cardviewshown':
         case 'cardviewclosed':
           return 'cardview';
+        case 'appwillopen':
+        case 'stackchanged':
+          return 'appopening';
+        case 'appwillclose':
+        case 'stackchanged':
```

Comment:
not sure I follow you...
you have 2 "groups" causing the statubar to stop, appopening and appclosing.
For each group you need an event (or more) that paused and one (or more) that resumes. But you can't have an event in 2 groups...